### PR TITLE
Update non-delivery-reports-in-exchange-online.md

### PR DESCRIPTION
**Pull request by arindamthokder**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md [hunk 1]
```diff
@@ -22,6 +22,17 @@ description: "Admins can learn about non-delivery reports (also known as NDRs or
 
 When there's a problem delivering an email message that you sent, Microsoft 365 or Office 365 sends an email to let you know. The email you receive is a delivery status notification, also known as a DSN or bounce message. The most common type is called a non-delivery report (NDR) and they tell you that a message wasn't delivered. Non-delivery can be caused by something as simple as a typo in an email address. NDRs include an error code that indicates why your email wasn't delivered, solutions to help you get your email delivered, a link to more help on the web, and technical details for administrators. Find out [What's included in an NDR?](#whats-included-in-an-ndr).
 
+## Run Non-delivery report (NDR) diagnostics
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md:25]:
```suggestion
## Run non-delivery report diagnostics
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md [hunk 2]
```diff
@@ -22,6 +22,17 @@ description: "Admins can learn about non-delivery reports (also known as NDRs or
...
+
+You can run an automated diagnostic to know more about the decription of the NDR, possible cause and solution by running the following NDR diagnostic. Please find the NDR code or Status code from the Undeliverable/Non-Delivery Report.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md:27]:
```suggestion
You can run an automated diagnostic to know more about the description of the NDR, possible cause, and solution by running the following NDR diagnostic. Find the NDR code or status code from the undeliverable/non-delivery report. 
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md [hunk 3]
```diff
@@ -22,6 +22,17 @@ description: "Admins can learn about non-delivery reports (also known as NDRs or
 
 When there's a problem delivering an email message that you sent, Microsoft 365 or Office 365 sends an email to let you know. The email you receive is a delivery status notification, also known as a DSN or bounce message. The most common type is called a non-delivery report (NDR) and they tell you that a message wasn't delivered. Non-delivery can be caused by something as simple as a typo in an email address. NDRs include an error code that indicates why your email wasn't delivered, solutions to help you get your email delivered, a link to more help on the web, and technical details for administrators. Find out [What's included in an NDR?](#whats-included-in-an-ndr).
 
+## Run Non-delivery report (NDR) diagnostics
+
+You can run an automated diagnostic to know more about the decription of the NDR, possible cause and solution by running the following NDR diagnostic. Please find the NDR code or Status code from the Undeliverable/Non-Delivery Report. 
+
+To run the diagnostic check, click the button below.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md:29]:
```suggestion
Find the NDR code or status code from the undeliverable/non-delivery report.  To run the diagnostic check, click the following button: 
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md [hunk 4]
```diff
@@ -22,6 +22,17 @@ description: "Admins can learn about non-delivery reports (also known as NDRs or
...
+## Run Non-delivery report (NDR) diagnostics
+
+You can run an automated diagnostic to know more about the decription of the NDR, possible cause and solution by running the following NDR diagnostic. Please find the NDR code or Status code from the Undeliverable/Non-Delivery Report. 
+
+To run the diagnostic check, click the button below. 
+
+> [!div class="nextstepaction"]
+> [Run Tests: NDR diagnostics](https://aka.ms/PillarEmailNDR)
+
+A flyout page opens in the Microsoft 365 admin center. Paste the NDR code or eror message and and click **Run Tests**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md:34]:
```suggestion
A flyout page opens in the Microsoft 365 admin center. Paste the NDR code or error message and select **Run Tests**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arindamthokder, kindly commit the suggested changes. Many thanks.

**Comment by arindamthokder** [comment]
> Copy review - @arindamthokder, kindly commit the suggested changes. Many thanks.

done, thanks a lot.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by arindamthokder** [comment]
Is there anything pending on this change? I do not see the update on the article?

**Comment by arindamthokder** [comment]
> @tiburd
Is there anything pending on this change? I do not see the update on the article?

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md [hunk 5]
```diff
@@ -22,6 +22,17 @@ description: "Admins can learn about non-delivery reports (also known as NDRs or
 
 When there's a problem delivering an email message that you sent, Microsoft 365 or Office 365 sends an email to let you know. The email you receive is a delivery status notification, also known as a DSN or bounce message. The most common type is called a non-delivery report (NDR) and they tell you that a message wasn't delivered. Non-delivery can be caused by something as simple as a typo in an email address. NDRs include an error code that indicates why your email wasn't delivered, solutions to help you get your email delivered, a link to more help on the web, and technical details for administrators. Find out [What's included in an NDR?](#whats-included-in-an-ndr).
 
+## Run non-delivery report diagnostics
+
+You can run an automated diagnostic to find out more about the description of the non-delivery report (NDR), possible cause, and solution by running the following NDR diagnostic. Make sure you get the NDR code or status code from the undeliverable/non-delivery report.
```
Comment (tiburd) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md:27]:
```suggestion
You can run an automated diagnostic to learn more about the description of the non-delivery report (NDR), possible cause, and solution by running the following NDR diagnostic. Make sure you get the NDR code or status code from the undeliverable/non-delivery report. 
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md [hunk 6]
```diff
@@ -22,6 +22,17 @@ description: "Admins can learn about non-delivery reports (also known as NDRs or
...
+## Run non-delivery report diagnostics
+
+You can run an automated diagnostic to find out more about the description of the non-delivery report (NDR), possible cause, and solution by running the following NDR diagnostic. Make sure you get the NDR code or status code from the undeliverable/non-delivery report. 
+
+To run the diagnostic check, select the following button: 
+
+> [!div class="nextstepaction"]
+> [Run Tests: NDR diagnostics](https://aka.ms/PillarEmailNDR)
+
+A flyout page opens in the Microsoft 365 admin center. Paste the NDR code or error message and select **Run Tests**.
```
Comment (tiburd) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/non-delivery-reports-in-exchange-online.md:34]:
```suggestion
A flyout page opens in the Microsoft 365 admin center. Paste the NDR code or error message, and then select **Run Tests**.
```

**Comment by tiburd** [comment]
@arindamthokder 

Thank you for pinging me - I apologize that this PR slipped through the cracks earlier. I've merged the PR - the changes should live shortly after 10:30 AM Pacific time today.

This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=main). We reviewed and merged this PR but encourage you to use the private repo for future contributions to Microsoft Docs.

Thanks!

**Pull request merged**